src/mjolnir/adminbuilder.cc

```diff
@@ -14,6 +10,11 @@
 #include <boost/property_tree/ptree.hpp>
 
 #include <geos_c.h>
+#include <sqlite3.h>
+
+#include <cstdint>
+#include <string>
+#include <vector>
 
 BOOST_GEOMETRY_REGISTER_POINT_2D(valhalla::midgard::PointLL,
```

Comment:
when i refactored this to use geos originally i was really hoping to get back and use geos for the whole rest of it and completely remove boost from the picture. we'll get to it one day :smile: